app/views/admin/parliaments/_form.html.erb

```diff
@@ -14,6 +14,9 @@
       <%= form.submit 'Send dissolution emails', name: 'send_emails', class: 'button-secondary', data: { confirm: 'Email everyone about dissolution?' } %>
     <% end %>
   <% end %>
+  <% if Archived::Petition.can_anonymize? %>
+    <%= form.submit 'Anonymize petitions', name: 'anonymize_petitions', class: 'button-secondary', data: { confirm: 'Anonymize all archived petitions?' } %>
```

Comment:
Supposedly the alert/confirm/prompt functions are to be removed from Chrome at some point - no idea how we'll do this after that 🤷🏻‍♂️